<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>993</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>941</width>
      <height>651</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
     <item>
      <widget class="PlotWidget" name="graphWidget" native="true"/>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Fit parameters</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>3</x>
          <y>10</y>
          <width>221</width>
          <height>581</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="7" column="0">
          <widget class="QCheckBox" name="ErosionCoefficientCheckbox">
           <property name="text">
            <string>ERCO</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="InitialMassCheckbox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>M0</string>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QCheckBox" name="ErosionMassMinCheckbox">
           <property name="text">
            <string>ERMm</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="AblationCoefficientInput"/>
         </item>
         <item row="9" column="1">
          <widget class="QLineEdit" name="ErosionMassMinInput"/>
         </item>
         <item row="7" column="1">
          <widget class="QLineEdit" name="ErosionCoefficientInput"/>
         </item>
         <item row="5" column="0">
          <widget class="QCheckBox" name="AblationCoefficientCheckbox">
           <property name="text">
            <string>ABL</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="InitialMassInput"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="InitialVelocityInput"/>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="ZenithInput"/>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="DensityInput"/>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="InitialVelocityCheckbox">
           <property name="text">
            <string>V0</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="ErosionHeightInput"/>
         </item>
         <item row="8" column="1">
          <widget class="QLineEdit" name="MassIndexInput"/>
         </item>
         <item row="6" column="0">
          <widget class="QCheckBox" name="ErosionHeightCheckbox">
           <property name="text">
            <string>ERHT</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="DensityCheckbox">
           <property name="text">
            <string>DENS</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QCheckBox" name="MassIndexCheckbox">
           <property name="text">
            <string>ER_S</string>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QLineEdit" name="ErosionMassMaxInput"/>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="ZenithCheckbox">
           <property name="text">
            <string>Zc</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QCheckBox" name="ErosionMassMaxCheckbox">
           <property name="text">
            <string>ERMM</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="InitialMassValue">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="InitialVelocityValue">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="ZenithValue">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="DensityValue">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLabel" name="AblationCoefficientValue">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QLabel" name="ErosionHeightValue">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QLabel" name="ErosionCoefficientValue">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="8" column="2">
          <widget class="QLabel" name="MassIndexValue">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="9" column="2">
          <widget class="QLabel" name="ErosionMassMinValue">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="10" column="2">
          <widget class="QLabel" name="ErosionMassMaxValue">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="ComputationEnabledLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>580</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Recomputation Enabled</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="CoordinatesLabel">
        <property name="geometry">
         <rect>
          <x>16</x>
          <y>610</y>
          <width>201</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>993</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
